core/vm/eips.go

```diff
@@ -154,7 +153,9 @@ func enable2929(jt *JumpTable) {
 // - Reduces max refunds to 20% gas
 func enable3529(jt *JumpTable) {
 	jt[SSTORE].dynamicGas = gasSStoreEIP3529
-	jt[SELFDESTRUCT].dynamicGas = gasSelfdestructEIP3529
+
+	// SELFDESTRUCT is disabled in Scroll
```

Comment:
ditto

Reply:
The lines commented out are part of `EIP-2929` and `EIP-3529`. I think it's better to be explicit about disabling them here.